apiserver/application/application_test.go

```diff
@@ -1480,6 +1480,91 @@ func (s *applicationSuite) TestApplicationUpdateInvalidApplication(c *gc.C) {
 	c.Assert(err, gc.ErrorMatches, `application "no-such-application" not found`)
 }
 
+func (s *applicationSuite) TestApplicationUpdateSeries(c *gc.C) {
```

Comment:
removed changes to application_test.go and added unit tests to application_unit_tests.go